authpass/lib/ui/screens/entry_details.dart

```diff
@@ -270,6 +272,14 @@ class _EntryDetailsState extends State<EntryDetails>
     with StreamSubscriberMixin {
   List<Tuple3<GlobalKey<_EntryFieldState>, KdbxKey, CommonField?>>? _fieldKeys;
 
+  static final _zxcvbn = Zxcvbn();
+
+  static Result? _strength;
+
+  static void _set_strength(Result? result) {
+    _strength = result;
+  }
```

Comment:
This looks wrong.. never use static variables for state. There has to be a better way for communicating with the child widget.

I also don't quite understand it, as there can be multiple passwords in one entry.. which one will this show the strength of? is not really obvious to me. I think this should all be encapsulated in the password field.